dolphinscheduler-python/pydolphinscheduler/DEVELOP.md

```diff
@@ -83,6 +83,23 @@ black .
 flake8
 ```
 
+## Build Docs
+We use [sphinx][sphinx] to build docs. DS Python API CI would automatically build docs when you submit pull request in 
+GitHub. You may locally ensure docs could be built suceessfully in case the failure blocks CI.
+
+To build docs locally, install sphinx and related python modules first via:
+```shell
+pip install sphinx
+pip install sphinx_rtd_theme
+pip install sphinx_click 
```

Comment:
we have a shortcut in `setup.py`. which you could find in https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-python/pydolphinscheduler/setup.py#L175
```suggestion
pip install .[doc]
```

Reply:
In that case, will change it to `pip install .[doc]`, thx for the suggestions!